testing/fil_token_integration/actors/basic_token_actor/src/lib.rs

```diff
@@ -144,13 +145,30 @@ impl Cbor for MintReturn {}
 
 impl BasicToken<'_> {
     fn mint(&mut self, params: MintParams) -> Result<MintReturn, RuntimeError> {
-        self.util.mint(
+        // Token::mint shouldn't save state at all
+        //let old_cid = sdk::sself::root().unwrap();
```

Comment:
Delete these lines. The comment is a bit off given the saving at line 159. But I agree we don't need the old CID, just return an error.